scripts/build_twilio_library.py

```diff
@@ -30,7 +30,10 @@ def generate(openapi_spec_path: str, output_path: str, language: str, domain: st
 
     to_generate = 'terraform-provider-twilio' if language == 'terraform' else f'twilio-{language}'
     is_domain_irrelevant = language in {'go', 'terraform'} and domain_name == 'preview'
-    sub_dir = subdirectories.get(language, 'rest')
```

Comment:
`subdirectories` is a mapping of which libraries use a sub-dir that's not `rest`. Should add an override there for `csharp` instead of logic here.

Reply:
Added new entry for csharp language in subdirectories map and removed extra logic. @childish-sambino Can you please review if this looks good now?